pyomo/contrib/pynumero/examples/callback/cyipopt_callback_halt.py

```diff
@@ -0,0 +1,24 @@
+import pyomo.environ as pyo
+from pyomo.contrib.pynumero.examples.callback.reactor_design import model as m
+
+"""
+This example uses an iteration callback to halt the solver
+"""
+def iteration_callback(nlp, alg_mod, iter_count, obj_value,
+                       inf_pr, inf_du, mu, d_norm, regularization_size,
+                       alpha_du, alpha_pr, ls_trials):
+    if iter_count >= 4:
+        return False
+    return True
```

Comment:
(comment/question) Is it any more clear to just make this
```python
return iter_count < 4
```

Reply:
Nope :)

Reply:
At least not to me :)